### PR TITLE
Match style with current homepage

### PR DESCRIPTION
**Pull request by limsohee1002**

### CODE REVIEW

File: src/components/BlockImage.js [hunk 1]
```diff
@@ -26,10 +27,19 @@ const BlockImage = props => {
...
+      })}
     >
       <div
-        className={`BlockImage__image-container BlockImage__image-container--${imageSize}-${imageAlign} overflow-hidden flex items-center justify-center`}
+        className={cx(`BlockImage__image-container overflow-hidden`, {
+          'md:col-12': imageSize === 'full',
+          'md:col-6': imageSize === 'large',
+          'md:col-5': imageSize === 'medium',
+          'md:col-4': imageSize === 'small'
+        })}
```
Comment (joshiefishbein) [inline src/components/BlockImage.js]: I see you're using backticks, single quotes, and double quotes all the same way in a matter of 7 lines lol. I recommend sticking with single quotes or installing prettier to do that for ya :P

File: src/components/WorkSection.js [hunk 2]
```diff
@@ -112,10 +92,12 @@ class WorkSection extends PureComponent {
   };
 
   renderWork = () => {
-    const { activeIndex, slideCount, mediaDimensions, isMobile } = this.state;
+    const { activeIndex, slideCount, mediaDimensions } = this.state;
     const activeProject = get(this, "props.selectedWorks")[activeIndex];
```
Comment (joshiefishbein) [inline src/components/WorkSection.js]: why would `activeIndex` not be part of the getter????

```typescript
get(this, `props.selectedWorks[${activeIndex}]`)
```

File: src/components/base/Markdown.js [hunk 3]
```diff
@@ -3,14 +3,15 @@ import PropTypes from 'prop-types';
 
 import Remarkable from 'react-remarkable';
 
-const Markdown = ({ src }) => (
-  <div className="Markdown">
+const Markdown = ({ src, fontSize = '' }) => (
+  <div className={`Markdown Markdown--${fontSize}`}>
```
Comment (joshiefishbein) [inline src/components/base/Markdown.js]: let's use a cx conditional so we don't end up with `Markdown--` when `fontSize` defaults to `''`

File: src/components/BlockImage.js (hunk 1 above)
Comment (limsohee1002) [reply src/components/BlockImage.js]: oh, I thought there is prettier already so I didn't pay attention too much! I will install it!

**Pull request merged**